lua/telescope/pickers.lua

```diff
@@ -1540,6 +1541,33 @@ function Picker:_get_next_filtered_prompt()
   return prompt
 end
 
+function Picker:_handle_line_number(prompt)
```

Comment:
Just an idea, I can help on that or iterate after. But what do you think about adding column here as well? It will be able to handle the paths output from the compiler that looks like 

```
/some/path/file.rs:121:64
```

Reply:
I think that'll be better. to me, however, it's enough to just jump to a specific line.

Reply:
like i already said [here](https://github.com/nvim-telescope/telescope.nvim/issues/2650) i kinda dont want this, but it also dependens on the complexity of the resulting code. So i'll do my final judgement once i see the code.